db/migrate/20200406213624_create_user.rb

```diff
@@ -0,0 +1,13 @@
+class CreateUser < ActiveRecord::Migration[5.1]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :address
+      t.string :city
+      t.string :state
+      t.string :zip
+      t.string :email
+      t.string :password_digest
+    end
```

Comment:
I am still unclear whether we need to explicitly add `t.timestamps` but Users definitely should have them... Can anyone confirm if this is a necessity or if rails adds them by default?

Reply:
@tony-gomes I think that is a group preference thing. I don't see timestamps being used in any of the user stories, so I think it's up to us to decide if they would be useful to store in our database.

Reply:
From what I understand, timestamps are something that we've always put in but haven't used so far, so I would expect it's something we should always put in